common/web/types/src/kmx/kmx-plus/kmx-plus.ts

```diff
@@ -197,7 +192,7 @@ export class Strs extends Section {
   }
 
   /** process everything according to opts */
-  static processString(s: string, opts: StrsOptions, sections: DependencySections) {
+  processString(s: string, opts: StrsOptions, sections: DependencySections) {
```

Comment:
wasn't called anywhere else, didn't need to be static

Reply:
```suggestion
  private processString(s: string, opts: StrsOptions, sections: DependencySections) {
```

The nice thing that `static` gives us is it tells us it has no access to member properties or functions, so it's more likely to be pure. Which of course, it now isn't :grin:: it now has a side-effect, which we really should document.